lib/tsconfig/3.8.x/tsconfig.json

```diff
@@ -0,0 +1,67 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "allowUmdGlobalAccess": false,
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "alwaysStrict": true,
+    "charset": "utf8",
+    "declaration": true,
+    "diagnostics": false,
+    "downlevelIteration": true,
+    "emitBOM": false,
+    "emitDecoratorMetadata": true,
+    "emitDeclarationOnly": false,
+    "experimentalDecorators": true,
+    "forceConsistentCasingInFileNames": false,
+    "importHelpers": true,
+    "incremental": true,
+    "inlineSourceMap": false,
+    "inlineSources": true,
+    "isolatedModules": false,
+    "listEmittedFiles": false,
+    "listFiles": false,
+    "module": "es2020",
+    "moduleResolution": "node",
+    "noEmitHelpers": false,
+    "noEmitOnError": false,
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitUseStrict": false,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "preserveConstEnums": true,
+    "pretty": true,
+    "removeComments": false,
+    "sourceMap": true,
+    "strict": true,
+    "strictBindCallApply": true,
+    "strictFunctionTypes": true,
+    "strictPropertyInitialization": true,
+    "strictNullChecks": true,
+    "stripInternal": true,
+    "skipLibCheck": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "suppressExcessPropertyErrors": false,
+    "useDefineForClassFields": true,
+    "target": "es2020",
```

Comment:
Same remark: Why is this done already in this config for TS 3.8? As far as I can see, this is suggested as from Angular 10: see [Why "es2020" instead of "esnext"?](https://angular.io/guide/migration-update-module-and-target-compiler-options#why-es2020-instead-of-esnext)

Reply:
Actually, I based this change on TypeScript release notes: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#es2020-for-target-and-module 
But I can revert if you would prefer 😊

Reply:
Ah ok, I missed that. So I guess it should be ok :wink: :+1: